ext/ddtrace_profiling_native_extension/extconf.rb

```diff
@@ -122,6 +140,7 @@ def add_compiler_flag(flag)
 
 # If we got here, libddprof is available and loaded
 ENV['PKG_CONFIG_PATH'] = "#{ENV['PKG_CONFIG_PATH']}:#{Libddprof.pkgconfig_folder}"
```

Comment:
Can `ENV['PKG_CONFIG_PATH']` be `nil`, or `''`? This would result in a leading `:`.

Reply:
Yes, it can. That's actually what happens almost always, since nothing in Ruby sets `PKG_CONFIG_PATH` by default. I can confirm `pkg-config` is OK with having the leading `:`.